types/twitch-ext/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for non-npm package twitch-ext 1.15
+// Type definitions for non-npm package twitch-ext 1.16
```

Comment:
```suggestion
// Type definitions for non-npm package twitch-ext 1.20
```
This should ideally mirror the major.minor version hosted at https://extension-files.twitch.tv/helper/v1/twitch-ext.min.js. It's embedded there somewhat opaquely, but the current version is `1.20.0` (grep for `environment:"production",version`).